src/Symfony/Bridge/Twig/Tests/Extension/WorkflowExtensionTest.php

```diff
@@ -38,7 +38,7 @@ protected function setUp()
         $workflow = new Workflow($definition);
 
         $registry = new Registry();
-        $registry->add($workflow, new ClassInstanceSupportStrategy(\stdClass::class));
+        $registry->addWorkflow($workflow, new ClassInstanceSupportStrategyWorkflow(\stdClass::class));
```

Comment:
I think we should keep a test case for the deprecated method to make sure we don't break it (flagging it as legacy then of course)